README.md

```diff
@@ -76,6 +76,24 @@ cmake --build .
 ctest -a --output-on-failure .
 ```
 
+## Install artifacts
+
+By default the reference client (`ndt7-client-cc`) and the single include
+libndt7.hpp are installed to system a default location like `/usr/local/`.
+
+```sh
+$ cmake --install .
+-- Install configuration: ""
+-- Installing: /usr/local/bin/ndt7-client-cc
+-- Installing: /usr/local/include/libndt7/libndt7.hpp
+```
+
+You may specify an alternate prefix.
+
+```sh
+cmake --install . --prefix </my/install/prefix>
```

Comment:
```suggestion
cmake --install . --prefix /my/install/prefix
```

I'd remove the `<` and `>` because the `sh` parser for the markdown would show them strangely, since `<` and `>` have the meaning of redirection when using the `sh` syntax highlighter.